README.md

```diff
@@ -211,3 +211,9 @@ define_subclasses(definitions, BRICK.Equipment)
 ```
 
 For now, the code is the documentation. Look at `equipment.py`, `point.py`, etc for examples and how to add to each of the class hierarchies.
+
+
+## How to Test?
+1. ``python tests/test_inference.py``: A basic test if Classes can be inferred from associated Tags.
+2. ``python tests/test_hierarchy_inference.py``: It tests whether all the instances associated with the right tags are correctly inferred through the hierarchy. For example, an instance with Tags of ``tag:Temperature`` and ``tag:Sensor`` should be inferred as ``brick:Temperature_Sensor``, ``brick:Sensor``, and ``brick:Point``.
+    - This test involves a time-consuming reasoning process, which took 4.5 hours in my desktop. In case you have the result graph at ``tests/test_hierarchy_inference.ttl``, you can reuse the file with an option like ``python tests/test_hierarchy_inference.py --reuse-inference``.
```

Comment:
Can you try using `owlready2` as the reasoner? Its much faster and shouldn't need the flag for re-using the inference

Reply:
I think these instructions are misleading. Tests are run using `pytest`, not by executing the scripts directly. If we want to have a list of tests, we can have a subsection under `## Tests` in the file.

Reply:
- Yeah, the README file should be updated. I will fix it soon.
- I tested owlready2 but the reasoning completeness is different than owlrl's. It wasn't able to properly infer superclasses. I think owlready2 is faster because its reasoning logic is simpler.

Reply:
- Updated README file.
- Please verify my observation of the difference between owlready2 and owlrl.

Reply:
When the owlrl inference finishes, I'll let you know :)

I can't find a good definition of the differences between them, but I've been developing using owlready2 and its worked great for me so far. Do you have an example of inference that it doesn't perform?

Reply:
owlready2 cannot infer an entity's parent classes. For example, given
```turtle
ex:space_0    brick:hasTag    tag:Space.
ex:space_0    brick:hasTag    tag:Location.
```
, owlready2 can infer like

```turtle
ex:space_0    a    brick:Space.
```
but not 
```turtle
ex:space_0    a    brick:Space, brick:Location.
```

Basically, I need RDFS inference in the test.

Reply:
Ok, I can reproduce this. I agree we need to use owlrl to do the inference properly for now. It makes the build time longer, which is unfortunate. To mitigate that, I was thinking that we commit the fully materialized ontology after running it through the reasoner.

This doesn't actually save us time though, because we'll ultimately have to run it for each of the tests independently (requiring different runs of the reasoner) *or* write the tests so that they can all operate on the same graph (which makes tests harder to write).

instead, if we organize test files by tests that operate on the same graph -- as in, the set of triples added at the beginning of the test file -- then we can somewhat amortize the cost of running the reasoner.

What do you think?